src/pages/CytAssist.tsx

```diff
@@ -0,0 +1,337 @@
+import MutedText from '../components/MutedText';
+import ScanInput from '../components/scanInput/ScanInput';
+import { Select } from '../components/forms/Select';
+import { objectKeys } from '../lib/helpers';
+import Label from '../components/forms/Label';
+import Table, { TableBody, TableCell, TableHead, TableHeader } from '../components/Table';
+
+/**
+ * Success notification when slots have been copied
+ */
+const ToastSuccess = () => <Success message={'Slots copied'} />;
+
+interface OutputLabwareScanPanelProps {
+  preBarcode: string | undefined;
+  onChangeBarcode: (barcode: string) => void;
+  onChangeLabwareType: (labwareType: string) => void;
+  onChangeCosting: (costing: string) => void;
+}
+
+/**Component to configure the output CytAssist labware**/
+const CytAssistOutputlabwareScanPanel: React.FC<OutputLabwareScanPanelProps> = ({
+  preBarcode,
+  onChangeBarcode,
+  onChangeLabwareType,
+  onChangeCosting
+}) => {
+  /**State to store preBarcode validation errors**/
+  const [preBarcodeValidationError, setPreBarcodeValidationError] = React.useState('');
+  return (
+    <div className={'w-full flex flex-row space-x-6'}>
+      <div data-testid="external-barcode">
+        <Label name={'External barcode'} />
+        <ScanInput
+          onScan={(value) => {
+            if (value.length === 0) {
+              setPreBarcodeValidationError('Required field');
+              onChangeBarcode('');
+            } else {
+              const valid = /[A-Z]\d{2}[A-Z]\d{2}-\d{7}-\d{2}-\d{2}/.test(value);
+              if (!valid) {
+                setPreBarcodeValidationError('Invalid format');
+                onChangeBarcode('');
+              } else {
+                setPreBarcodeValidationError('');
+                onChangeBarcode(value);
+              }
+            }
+          }}
+          allowEmptyValue={true}
+          placeholder={'V42A20-3752023-10-20'}
+        />
+        {preBarcodeValidationError.length > 0 ? (
+          <MutedText className={'text-red-400'}>{preBarcodeValidationError}</MutedText>
+        ) : preBarcode === undefined || preBarcode.length <= 0 ? (
+          <MutedText>Input barcode and press 'Enter' key</MutedText>
+        ) : (
+          ''
+        )}
+      </div>
+      <div>
+        <Label name={'Labware Type'} />
+        <Select
+          onChange={(e) => {
+            onChangeLabwareType(e.currentTarget.value);
+          }}
+          emptyOption={false}
+          data-testid="output-labware-type"
+        >
+          {objectKeys(LabwareTypeName)
+            .filter(
+              (key) =>
+                LabwareTypeName[key] === LabwareTypeName.VISIUM_LP_CYTASSIST ||
+                LabwareTypeName[key] === LabwareTypeName.VISIUM_LP_CYTASSIST_XL
+            )
+            .map((key) => (
+              <option key={key} value={LabwareTypeName[key]}>
+                {LabwareTypeName[key]}
+              </option>
```

Comment:
Think this filter to map logic is a bit of a code smell. Think you dont need either and could do just to options like `<option key={LabwareTypeName.VISIUM_LP_CYTASSIST} value= {LabwareTypeName.VISIUM_LP_CYTASSIST} >
{LabwareTypeName.VISIUM_LP_CYTASSIST}
</option>

<option key={LabwareTypeName.VISIUM_LP_CYTASSIST_XL} value= {LabwareTypeName.VISIUM_LP_CYTASSIST_XL} >
{LabwareTypeName.VISIUM_LP_CYTASSIST_XL}
</option>
`

Reply:
Thought it would make it less verbose and easy to modify, if  change in labware type in future.